hudi-common/src/main/java/org/apache/hudi/common/model/HoodieFileGroup.java

```diff
@@ -146,14 +157,14 @@ public Option<FileSlice> getLatestFileSlicesIncludingInflight() {
   /**
    * Provides a stream of committed file slices, sorted reverse base commit time.
    */
-  public Stream<FileSlice> getAllFileSlices() {
-    if (!timeline.empty()) {
+  public Stream<FileSlice> getAllCommittedFileSlices() {
```

Comment:
I don't think we need to fix the method name. we already have a method called isFileSliceCommitted. so that should suffice.